src/components/Composer.vue

```diff
@@ -421,7 +422,11 @@ export default {
 				.then(this.getMessageData())
 				.then(data => this.send(data))
 				.then(() => logger.info('message sent'))
-				.then(() => (this.state = STATES.FINISHED))
+				.then(() => {
+					emit('mail:interaction', {type: 'message-sent', recipient: this.selectedUser.user})(
```

Comment:
:thinking: These are two statements. the `emit` call and the assignment. You create a very creative syntax :)

Just make it
```
.then(() => {
  emit('mail:interaction', {type: 'message-sent', recipient: this.selectedUser.user})
  this.state = STATES.FINISHED
}
```

Reply:
thats what i did, but with bracket :P i like bracket

Reply:
the problem is that you did `emit(...)(...)` which means you treated the return value of `emit(...)` as a function which you ten called :wink: